examples/deployments/terraform-gcp-github-actions/variables.tf

```diff
@@ -0,0 +1,36 @@
+locals {
+  labels = {
+    "service" = var.service
+  }
+}
+
+variable "project" {
+  type        = string
+  description = "ID Google project"
+}
+
+variable "region" {
+  type        = string
+  description = "Region Google project"
+}
+
+variable "service" {
```

Comment:
Does it make sense to have a sensible default here?

Reply:
e.g. `chroma-service`

Reply:
yeah I think so, added it and used it